addons/stock/data/stock_data.xml

```diff
@@ -118,9 +118,14 @@
         <function model="res.company" name="create_missing_scrap_location"/>
         <function model="res.company" name="create_missing_scrap_sequence"/>
 
+        <!-- Config Parameters -->
         <record model="ir.config_parameter" id="barcode_separator" forcecreate="False">
             <field name="key">stock.barcode_separator</field>
             <field name="value">,</field>
         </record>
+        <record model="ir.config_parameter" id="visibility_days">
```

Comment:
In stable, this parameter won't exist for existing DB (no update data)
It is not an issue since its value is 0 and it's also the default value for when this parameter doesn't exist, so its presence is not important, it's just FYI